### ISSUE TITLE
Sync color scheme with vscode theme

### ISSUE THREAD

**Issue by gudvinr** [microsoft/vscode-livepreview#661]
Currently, live preview browser sets `@media (prefers-color-scheme)` according to _system_ settings, not to whatever theme you use in vscode.

There's also no way to override this other than manually configuring that in devtools appearance settings.

So, if you use dark theme but system doesn't support color scheme or uses light theme, your webview will be using light color sheme.

Instead, there at least could be an option to set "system" appearance mode for the extension to match vscode theme setting. For example, VSCode itself can optionally inherit system settings if `window.autoDetectColorScheme` is enabled but it is not enforced.

Possible configuration options are:
* LP follows system theme (and ignores vscode theme)
* LP follows vscode theme (which _might_ follow system theme)
* LP prefers dark theme
* LP prefers light theme

**Comment by valler** [comment]
I disagree on this. It's frustrating enough that the preview is, as of now, affected by the VS Code font size. The preview should behave just like a normal browser except for hot reloading, so a dev sees what a user would see without being affected by the devs prefs for the IDE. If such a feature makes it into the extension, it should be opt in.

**Comment by kycutler** [comment]
The Live Preview extension now uses VS Code's Integrated Browser, which respects the `window.systemColorTheme` setting. You can set that to `auto` to follow the VS Code color scheme.
